passivbot.py

```diff
@@ -156,8 +158,14 @@ async def update_position(self) -> None:
                  (qty_to_cost(position['shrt']['size'], position['shrt']['price'],
                               self.xk['inverse'], self.xk['c_mult'])
                   if position['shrt']['price'] else 0.0)) / self.max_leverage
-            position['equity'] -= position['wallet_balance'] * (1 - self.cross_wallet_pct)
-            position['wallet_balance'] *= self.cross_wallet_pct
+
+            position['wallet_balance'] = (position['wallet_balance'] if self.assigned_balance is None
```

Comment:
@enarjord there is no check that the assigned_balance does not exceed the actual balance in the wallet, shouldn't there at least be some check on that?

Reply:
if other spot bots using same quote token have positions, bot doesn't know what correct wallet balance is

it only sees quote balance onhand

Reply:
if multiple bots use the same asset (for which this PR is a use case), then you are right. Perhaps we don't need to shield people from their own stupidity, but if somebody makes a type (say they have a $1000 wallet, and they accidentally set assigned_balance to $5000 (extra 0)), the bot will happily think it has $5000 to work with. I'm happy to leave it as is, just something that came up